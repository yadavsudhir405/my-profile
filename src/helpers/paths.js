const paths = {
  default: '/',
  login: '/login',
};

export default paths;
