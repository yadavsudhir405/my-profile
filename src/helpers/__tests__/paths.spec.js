import paths from '../paths';

describe('Paths Helper', () => {
  it('should match snapshot', () => {
    expect(paths).toMatchSnapshot();
  });
});
