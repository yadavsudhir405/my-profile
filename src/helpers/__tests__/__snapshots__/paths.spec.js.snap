// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paths Helper should match snapshot 1`] = `
Object {
  "claims": "/claims",
  "default": "/health",
  "health": "/health",
  "login": "/login",
  "profile": "/profile",
  "shop": "/shop",
}
`;
